cosmos/lava.json

```diff
@@ -0,0 +1,51 @@
+{
+  "rpc": "https://lava.tendermintrpc.lava.build",
+  "rest": "https://lava.rest.lava.build",
+  "chainId": "lava-mainnet-1",
+  "chainName": "Lava",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/lava/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "LAVA",
+    "coinMinimalDenom": "ulava",
+    "coinDecimals": 6,
+    "coinGeckoId": "lava-network",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/lava/ulava.png"
+  },
+  "walletUrl": "https://lavanet.xyz",
```

Comment:
Could you remove this line as well? walletUrl is a deprecated property.

Reply:
done